src/Functions/formatDateTime.cpp

```diff
@@ -38,22 +38,19 @@ namespace ErrorCodes
 
 namespace
 {
+using Pos = const char *;
 
-struct FormatDateTimeTraits
+enum class SupportInteger
```

Comment:
```suggestion
enum class SupportInteger : uint8_t
```

Reply:
Not really sure about this. There are cases where the specification of the underlying type is crucial, e.g. if the enum values are persisted. Here, enum values merely exist at compile-time, and an explicit type is visual clutter, imho.